po/pl.po

```diff
@@ -0,0 +1,2133 @@
+# Polish translations for GrandOrgue package.
+# This file is put in the public domain.
+# Romuald Milewski <rommil67@gmail.com>, 2021
+msgid ""
+msgstr ""
+"Project-Id-Version: GrandOrgue\n"
+"Report-Msgid-Bugs-To: ourorgan-developers@lists.sourceforge.net\n"
+"POT-Creation-Date: 2013-02-18 09:03+0100\n"
+"PO-Revision-Date: 2013-02-16 13:30+0100\n"
+"Last-Translator: Romuald Milewski <rommil67@gmail.com>\n"
+"Language-Team: Polish <rommil67@gmail.com>\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid " (PA): "
+msgstr " (PA): "
+
+msgid " - Virtual Pipe Organ Software"
+msgstr " - Program wirtualnych organów piszczałkowych"
+
+msgid "\""
+msgstr "\""
+
+msgid "#"
+msgstr "#"
+
+#, c-format
+msgid "%.3f MB"
+msgstr "%.3f MB"
+
+#, c-format
+msgid "%.3f MB of %.3f MB"
+msgstr "%.3f MB von %.3f MB"
+
+#, c-format
+msgid "%d"
+msgstr "%d"
+
+#, c-format
+msgid "%d (%s)"
+msgstr "%d (%s)"
+
+#, c-format
+msgid "%f cent"
+msgstr "%f cent"
+
+#, c-format
+msgid "%s - left"
+msgstr "%s - lewy"
+
+#, c-format
+msgid "%s - right"
+msgstr "%s - prawy"
+
+#, c-format
+msgid "%s: %f dB"
+msgstr "%s: %f dB"
+
+#, c-format
+msgid "%s: mute"
+msgstr "%s: wycisz"
+
+msgid "&About"
+msgstr "&O"
+
+msgid "&Add"
+msgstr "&Dodaj"
+
+msgid "&Audio/Midi"
+msgstr "&Audio/Midi"
+
+msgid "&Channel:"
+msgstr "&Kanał:"
+
+msgid "&Close"
+msgstr "&Zamknij"
+
+msgid "&Convolution reverb"
+msgstr "&Pogłos"
+
+msgid "&Data:"
+msgstr "&Dane:"
+
+msgid "&Delete"
+msgstr "&Usuń"
+
+msgid "&Device:"
+msgstr "&Urządenia:"
+
+msgid "&Enhancements"
+msgstr "&Rozszerzenia"
+
+msgid "&Event:"
+msgstr "&Zdarzenie:"
+
+msgid "&Export Settings/Combinations"
+msgstr "&Eksport Ustaiwń/Kombinacji"
+
+msgid "&File"
+msgstr "&Plik"
+
+msgid "&Help"
+msgstr "&Pomoc"
+
+msgid "&Help\tF1"
+msgstr "&Pomoc\tF1"
+
+msgid "&Highest key:"
+msgstr "&Najwyższy klawisz:"
+
+msgid "&Import Settings"
+msgstr "&Import ustawień"
+
+msgid "&Listen for Event"
+msgstr "Czekaj na zdarzenie"
+
+msgid "&Lowest key:"
+msgstr "&Najniższy klawisz:"
+
+msgid "&Memory Level"
+msgstr "&Poziom pamięci"
+
+msgid "&Off Value:"
+msgstr "&Wartośc końcowa"
+
+msgid "&On value:"
+msgstr "&Wartość początkowa"
+
+msgid "&Open\tCtrl+O"
+msgstr "Otwóż\tStrg+O"
```

Comment:
Well, this has already been merged, but this one is just plainly wrong (should be `otwórz` instead of `otwórz`). And even in this very same file there are properly translated instances of "open", even right below this one in the menu:

![image](https://user-images.githubusercontent.com/3586746/137086628-007c2a81-3efd-4c38-8773-101df70ffdbb.png)

Reply:
@pawel-szajna Because this PR is already merged and closed, please make a new one.